starky/src/cross_table_lookup.rs

```diff
@@ -1019,34 +1037,35 @@ pub(crate) fn verify_cross_table_lookups_circuit<
     debug_assert!(ctl_zs_openings.iter_mut().all(|iter| iter.next().is_none()));
 }
 
-#[cfg(test)]
-pub(crate) mod testutils {
-    use std::collections::HashMap;
+#[cfg(debug_assertions)]
+pub mod testutils {
+    #[cfg(not(feature = "std"))]
```

Comment:
Can't you make `std` a required feature of `debug_assertions`? That is, is it really plausible that one would run `debug_assertions` in a no-std env?

Reply:
Can this be enforced at the Cargo.toml level? Or should I just change the `#[cfg(debug_assertions)]` by `#[cfg(all(debug_assertions, feature = "std"))]`?